nrpy/infrastructures/BHaH/seobnr/BOB_C_codegen_library.py

```diff
@@ -0,0 +1,180 @@
+"""
+Set up C function library for SEOBNR inspiral integrations.
+
+Author: Zachariah B. Etienne
```

Comment:
I think you'll want to change this.

Reply:
I also see myself as the sole author in other files. At least you @siddharth-mahesh  should list yourself as well, as sole author or coauthor as applicable.

Reply:
Hi Zach,

Thanks for pointing these out.
I've pushed the requested changes.